src/main/resources/config.patch.yml

```diff
@@ -1,4 +1,8 @@
 # Put patches for BetonQuest's config here. The syntax is documented in the docs/API/ConfigPatcher.md
+2.0.0.6:
+  - type: TYPE_TRANSFORM
+    key: hook.redischat
+    newType: boolean
```

Comment:
```suggestion
```
New keys are added to the config automatically.

Reply:
Sorry, the redundant patch is my fault...